postprocessing/visualization/receiver/src/View.py

```diff
@@ -201,6 +204,15 @@ def plot(self):
             p.plot(wf.time, waveform, label=str(nWf))
             p.set_xlabel('t (s)')
           p.set_ylabel(name)
+          #print L2 error
+          if nWf > 0 and not self.diff.isChecked():
+            time_union = numpy.union1d(wf.time, wf_ref.time)
+            wf_interp = scipy.interpolate.interp1d(wf.time, waveform)
+            ref_interp = scipy.interpolate.interp1d(wf_ref.time, wf_ref.waveforms[name])
+            wf_union = wf_interp(time_union)
+            ref_union = ref_interp(time_union)
+            error = numpy.sqrt(scipy.integrate.trapz((wf_union - ref_union)**2))
```

Comment:
Sorry for this late comment, but actually, you could have specified the x parameter of scipy.integrate.trapz.
Then you can compare the diff across several couples of simulations.
(else your diff is strongly dependent on the time samples in each simulation).